clouddq/main.py

```diff
@@ -272,25 +279,21 @@ def main(  # noqa: C901
     metadata: Optional[str],
     dry_run: bool,
     progress_watermark: bool,
+    target_bigquery_summary_table: str,
```

Comment:
this is an optional argument

Reply:
we should also assert that this summary table is in the same region as --gcp_region_id. I don't see the assert_dataset_is_in_region() method call in this main class...does your branch have the latest commit from the main branch?